brian2/devices/device.py

```diff
@@ -412,10 +414,10 @@ def spike_queue(self, source_start, source_end):
         # Use the C++ version of the SpikeQueue when available
         try:
             from brian2.synapses.cythonspikequeue import SpikeQueue
-            logger.info('Using the C++ SpikeQueue', once=True)
+            logger.diagnostic('Using the C++ SpikeQueue', once=True)
         except ImportError:
             from brian2.synapses.spikequeue import SpikeQueue
-            logger.info('Using the Python SpikeQueue', once=True)
+            logger.diagnostic('Using the Python SpikeQueue', once=True)
```

Comment:
We could potentially have a "performance checklist" that is either displayed by default at some time or can be requested. For example, if you run a simulation that is expected to take more than 10m and there are performance related problems we detected, we show the checklist.